base/shell/explorer/traywnd.cpp

```diff
@@ -524,7 +328,7 @@ class CTrayWindow :
     public IContextMenu
 {
     CStartButton m_StartButton;
-    CTrayShowDesktopButton m_ShowDesktopButton;
+    CTrayShowDesktopButton* m_ptShowDesktopButton;
```

Comment:
nit: use `p` instead of `pt` for a pointer.
`pt` is (now and then) used for a POINT or POINTL